app/contributing/community-expectations.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Community expectations
+purpoae: Explore Kong's contribution community expectations - learn how to engage, collaborate, and contribute to Kong's open source projects efficiently.
```

Comment:
```suggestion
description: Explore Kong's contribution community expectations - learn how to engage, collaborate, and contribute to Kong's open source docs project efficiently.
```
This page is specifically about the docs project. We can't really speak for other open-source projects at Kong.